patched-fonts/FiraCode/readme.md

```diff
@@ -51,6 +51,11 @@ See the [Reserved Font Name section][SIL-RFN] for additional information
   * If you are limited to monospaced fonts (because of your terminal, etc) then pick a font with the `'Mono'` suffix.
     * This denotes that the Nerd Font glyphs will be monospaced not necessarily that the entire font will be monospaced
 
+### Ligatures
```

Comment:
Make the changes in the [readme-per-directory-addendum.md](https://github.com/ryanoasis/nerd-fonts/blob/master/src/readme-per-directory-addendum.md) instead :smiley:

Reply:
:heavy_check_mark: